man.Rmd

```diff
@@ -161,51 +164,53 @@ The introduction provides a title, description, and, optionally, details, for th
     The title is shown in various function indexes and is what the user will usually see when browsing multiple functions.
 
 -   The **description** is taken from the next paragraph.
-    It's show at the top of documentation and should briefly describe the most important features of the function.
+    It's shown at the top of documentation and should briefly describe the most important features of the function.
 
 -   Additional **details** are anything after the description.
-    Details are optional, but can be any length so are useful if want to dig deep into some important aspect of the function.
+    Details are optional, but can be any length so are useful if you want to dig deep into some important aspect of the function.
 
 The following sections describe each component in more detail, and then discuss a few useful related tags.
 
 ### Title
 
-When figuring out what to use as a title, I think it's most important about the reference index.
-When someone is skimming the index, how will they know which functions will solve their current problem?
+When writing the title, it's useful to think about how it will appear in the reference index.
+When a user is skims the index, how will they know which functions will solve their current problem?
 This requires thinking about what your functions have in common (which doesn't need to be repeated in every title) and what is unique to that function (which should be highlighted in the title).
 
-As an example, take the titles of some of the key dplyr functions[^man-3]:
+As an example, take the titles of some of the key functions in dplyr[^man-3]:
 
 [^man-3]: Like all the examples, these might have changed a bit since we wrote this book, because we're constantly striving to do better.
     You might compare what's in the book to what we now use, and consider if you think if it's an improvement.
 
--   `mutate()`: Create, modify, and delete columns.
--   `summarise()`: Summarize each group to fewer rows.
--   `filter()`: Subset rows using column values.
--   `select()`: Subset columns using their names and types.
--   `arrange()`: Arrange rows by column values.
+-   `mutate()`: Create, modify, and delete columns
+-   `summarise()`: Summarize each group to fewer rows
+-   `filter()`: Subset rows using column values
+-   `select()`: Subset columns using their names and types
+-   `arrange()`: Arrange rows by column values
```

Comment:
I think it's worth leaving the trailing `.` off here, since that's what we want for the actual oxygen title.